apps/katib/upstream/installs/katib-external-db-with-kubeflow/db-manager-patch.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
```

Comment:
can you comment on how these files are different from https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/apps/katib/upstream/installs/katib-external-db?

if we need to have a separate directory, this can be an overlay inside this directory? https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/apps/katib/upstream/installs/katib-with-kubeflow

Reply:
Added comment:
```
# The katib-external-db manifest does not create the necessary katib ui virtual service and doesn not apply the configurations from katib-cert-manager, which the katib-with-kubeflow manifest does. 
# This manifest extends off of the katib-with-kubeflow manifest and applies the necessary patch for katib to use an external db
```

The katib-with-kubeflow manifest needs to be organized within a base directory for an overlay to be applied. Technically with the current layout, katib-external-db-with-kubeflow is an overlay which takes katib-with-kubeflow as a base.

Reply:
not sure why they have it this way, We should propose this change upstream itself